app/models/manageiq/providers/vmware/infra_manager/vim_connect_mixin.rb

```diff
@@ -2,8 +2,6 @@ module ManageIQ::Providers::Vmware::InfraManager::VimConnectMixin
   extend ActiveSupport::Concern
 
   def vim_connect(options = {})
-    options[:auth_type] ||= :ws
-
```

Comment:
Deleting this had me concerned because I wasn't sure how the standard infra_manager connect was working, but when the requested authtype is missing the authentication_mixin automatically falls back to the `default_authentication_type` which in this case is `:default`.  Removing this is fine now that hosts also use `:default`